### PR TITLE
- composer text view font using UIAppearance

### PR DESCRIPTION
**Pull request by kartikthapar**
Understandably this is a bit of a hack but this is all I got!

### CODE REVIEW

**Comment by kmahal** [comment]
Reviewing now, @kartikthapar.  Have you tested this implementation?  Does it solve your problem from https://github.com/layerhq/Atlas-iOS/issues/818?  Getting tests on this would be good too.

File: Code/Views/ATLMessageInputToolbar.m [hunk 1]
```diff
@@ -240,6 +240,18 @@ - (void)setRightAccessoryButtonDisabledColor:(UIColor *)rightAccessoryButtonDisa
...
 
+- (void)setRightAccessoryButtonTitleLabelFont:(UIFont *)rightAccessoryButtonTitleLabelFont
+{
+    _rightAccessoryButtonTitleLabelFont = rightAccessoryButtonTitleLabelFont;
+    [self.rightAccessoryButton.titleLabel setFont:rightAccessoryButtonTitleLabelFont];
+}
+
+- (void)setTextInputFont:(UIFont *)textInputFont
+{
+    [self.textInputView setFont:textInputFont];
```
Comment (kmahal) [inline Code/Views/ATLMessageInputToolbar.m]: you must set the property's value here too.

**Comment by kmahal** [comment]
commented.  You must also set the default values in the default initialization method `+ (void)initialize`.

**Comment by kartikthapar** [comment]
Ya its a pretty loose PR, was just trying to get my things working. I’ll have this updated with tests, etc in a while. Thanks.

— Kartik Thapar

**Comment by kartikthapar** [comment]
There are issues with the current upstream; so I have revoked the PR. I will resubmit this in due time. Thanks.

**Pull request closed without merging**